crates/fluvio/src/producer/memory_batch.rs

```diff
@@ -8,20 +8,39 @@ use fluvio_types::Timestamp;
 
 use super::*;
 
+pub enum MemoryBatchStatus {
```

Comment:
is this need to be public?

Reply:
I think so, we need it in others modules